rotkehlchen/chain/ethereum/modules/yearn/graph.py

```diff
@@ -0,0 +1,424 @@
+    from rotkehlchen.db.dbhandler import DBHandler
+
+log = logging.getLogger(__name__)
+
+QUERY_V2_VAULTS = (
+    """
+    {{
+    vaults {{
+        id
+        shareToken {{
+            symbol
+            name
+            decimals
+        }}
+        token {{
+            symbol
+            name
+            decimals
+            id
+        }}
+        apiVersion
+        activation
+        tags
+    }}
+    }}
+    """
+)
+
+QUERY_USER_DEPOSITS = (
+    """
+    {{
+    deposits(
+        where: {{
+            account_in: $addresses,
+            blockNumber_gte: $from_block,
+            blockNumber_lte: $to_block,
+        }}) {{
+        id
+        blockNumber
+        timestamp
+        tokenAmount
+        sharesMinted
+        transaction{{
+        hash
+        }}
+        vault {{
+        id
+        token {{
+            id
+            symbol
+        }}
+        shareToken {{
+            id
+            symbol
+        }}
+        }}
+    }}
+    }}
+    """
+)
+
+QUERY_USER_WITHDRAWLS = (
+    """
+    {{
+    withdrawals(
+        where: {{
+            account_in: $addresses,
+            blockNumber_gte: $from_block,
+            blockNumber_lte: $to_block,
+        }}) {{
+        id
+        tokenAmount
+        sharesBurnt
+        blockNumber
+        timestamp
+        transaction{{
+        hash
+        }}
+        vault {{
+        id
+        token {{
+            id
+            symbol
+        }}
+        shareToken{{
+            id
+            symbol
+        }}
+        }}
+    }}
+    }}
+    """
+)
+
+QUERY_USER_EVENTS = (
+    """
+    accounts(
+        where: {{
+            id_in: $addresses
+        }}) {{
+            id
+            deposits(where:{{blockNumber_gte: $from_block, blockNumber_lte: $to_block }}){{
+                id
+                blockNumber
+                timestamp
+                tokenAmount
+                sharesMinted
+                vault {{
+                    id
+                    shareToken {{
+                        id
+                        symbol
+                        name
+                    }}
+                    token {{
+                        id
+                        symbol
+                        name
+                    }}
+                }}
+            }}
+            withdrawals(where:{{blockNumber_gte: $from_block, blockNumber_lte: $to_block }}) {{
+                id
+                blockNumber
+                timestamp
+                tokenAmount
+                sharesBurnt
+                vault {{
+                    id
+                    shareToken {{
+                        id
+                        symbol
+                        name
+                    }}
+                    token {{
+                        id
+                        symbol
+                        name
+                    }}
+                }}
+            }}
+        }}
+    }}
+    """
+)
+
+
+class YearnV2Inquirer:
```

Comment:
Please change the name. This should be `YearnVaultsV2Graph` or something like that. I thought it had something to do with prices when I first saw this.